app/src/main/java/com/islaharper/dawnofandroid/di/DatabaseModule.kt

```diff
@@ -0,0 +1,27 @@
+package com.islaharper.dawnofandroid.di
+
+import android.content.Context
+import androidx.room.Room
+import com.islaharper.dawnofandroid.data.local.AndroidFlavourDb
+import com.islaharper.dawnofandroid.util.Constants
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
+object DatabaseModule {
+
+    @Provides
+    @Singleton
+    fun provideDatabase(@ApplicationContext context: Context): AndroidFlavourDb {
+        return Room.databaseBuilder(
```

Comment:
Why would we write a `AndroidFlavourDb.create` factory function and then not use it in the Dagger provides function?

Reply:
Updated commit... This was an artifact from writing unit test for the remote mediator